authors.json

```diff
@@ -0,0 +1,102 @@
+[
+    {
+        "author_order": 10000,
+        "email": "cjwright4242@gmail.com",
+        "github_id": "CJ-Wright",
+        "institution": "Columbia University",
+        "institution_address": "",
+        "institution_address_peerj": [
+            "Department of Applied Physics and Applied Mathematics, Columbia University, New York, NY 10027"
+        ],
+        "institution_address_siam": "New York, NY 10027",
+        "name": "Christopher J. Wright"
+    },
+    {
+        "author_order": 0,
+        "email": "sb2896@columbia.edu",
```

Comment:
Thanks for including me Chris.  My interest and contributions to this project have been in supporting, mentoring and encouraging, CJ and Justin, and more laterally facilitating the writing of the paper for what I have come to learn is an important and impactful project.  I fully support the project and hope that I can be a supporter and advocate moving forward, and possibly even a contributor if time and expertise permits.    The paper is important so that my funders know that their funds are producing products that are helping the scientific community that they support (and how!). The timing of the paper is also important as I am about to try and renew the grant that supported this work (deadline 1/26/19), so a paper submitted after that date that acknowledges the grant is much less helpful.   The grant acknowledgement is key, my co-authorship less so, so if people think that I have not contributed sufficiently to warrant coauthorship I would happily withdraw my name if that is the consensus, though I would be honored to be considered to be a contributing partner to the enterprise.

I think that this is an awesome project, and I want to continue to work in the background to support it.